### PR TITLE
french translation

### PR DESCRIPTION
**Pull request by xisd**
Sorry I made a mistake (merged 2 branchs that should not have been merged) that made me delete the repository and re-fork
So here is only the french translation (with update and fix of previously noted errors) 
Formspec modifications are separated in an other branch (and PR)
And I made a separate mod to add an "award board" node

### CODE REVIEW

File: locale/fr.txt [hunk 1]
```diff
@@ -0,0 +1,196 @@
+%d/%d chat messages = %d/%d messages du chat 
+%d/%d crafted = %d/%d craftés
+%d/%d deaths = %d/%d morts 
+%d/%d dug = %d/%d creusés
+%d/%d game joins = %d/%d connexions
+%d/%d placed = %d/%d placés
+%s (got) = %s (obtenu)
+%s: %s = %s : %s
+%s’s awards: = Les trophés de %s :
```
Comment (Ekdohibs) [inline locale/fr.txt]: trophées

File: locale/fr.txt [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+(Secret Award) = (Trophé Secret) 
+<achievement ID> = <réussite ID>
+<name> = <nom>
+A Cat in a Pop-Tart?! = Un Chat dans un Pop-Tart ?!
+Achievement Unlocked! = Réussite Débloquée !
+Achievement Unlocked: = Réussite Débloquée :
+Achievement Unlocked: %s = Réussite Débloquée : %s
+Achievement not found. = Réussite non trouvée.
+All your awards and statistics have been cleared. You can now start again. = Tous vos trophés et statistiques ont été réinitialisés. Vous pouvez recommencer a nouveau.
+Aspiring Farmer = Apprenti.e. Fermière/Fermier
```
Comment (Ekdohibs) [inline locale/fr.txt]: Apprenti·e Fermi·er·ère

File: locale/fr.txt [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+Awards = Trophés
+Backpacker = Randonneuse/Randonneur
```
Comment (Ekdohibs) [inline locale/fr.txt]: Randonneu·r·se

File: locale/fr.txt [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+Bankier = Banquière/Banquier
+Bricker = Briqueuse/Briqueur
+Build a Cave = Construit une grotte
```
Comment (Ekdohibs) [inline locale/fr.txt]: Construis si on utilise l'impératif, sinon, mettre des infinitifs partout.

File: locale/fr.txt [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+Craft 400 yellow dyes. = Prépare 400 teintures jaunes.
+Craft 7 bookshelves. = Fabrique 7 bibliothèques
+Craft 8 times flint and steel. = Fabrique 8 silex et aciers.
+Craft 800 stone bricks. = Confectionne 800 briques de pierre.
+Craft a diamond block. = Confectionne un bloc de diamant.
+Craft: %d×%s = Fabrique : %d×%s 
+Craft: %s = Fabrique : %s 
+Crafter of Sticks = Fabriquant.e. de Bâtons.
+Dandelions are Yellow = Les Pissenlits sont Jaunes.
+Desert Discoverer = Expolratrice/Expolrateur du Désert.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Explorat·eur·rice du Désert

File: locale/fr.txt [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+Desert Dweller = Habitant.e. du Désert.
+Die %d times. = Mort.e. %d fois.
+Die. = Mort.e.
+Dig 1,000 copper ores. = Creuse 1,000 minerais de cuivre.
```
Comment (Ekdohibs) [inline locale/fr.txt]: 1000 ou 1 000, mais pas de virgule.

File: locale/fr.txt [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+Field Worker = Travailleuse/Travailleur Agricole
+Filthy Rich = Richissime
+Firefighter = Pompier
+First Day in the Woods = Premier Jour dans les Bois
+First Gold Find = Première Pépite : Or
+First Mese Find = Première Pépite : Mese
+Fortress = Forteresse
+Geraniums are Blue = Les Geraniums Sont Bleus
+Get the achievements statistics for the given player or yourself = Obtenez les statistiques des réussites d'un joueur donné ou de vous-même
+Girl's Best Friend = Girl's Best Friend
```
Comment (Ekdohibs) [inline locale/fr.txt]: Pourquoi ceci n'est pas traduit ?

File: locale/fr.txt [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+Glacier Discoverer = Découvreuse/Découvreur de glacier
+Glasser = Verrière/Verrier
+Gold Rush = Ruée vers l'or
+Grasslands Discoverer = Découvreuse/Découvreur de prairies
+Hardened Miner = Mineuse/Mineur Endurci.e.
+Hardest Block on Earth = Le Bloc le Plus Dur du Monde
+Harvest 125 fully grown wheat plants. = Récolte 125 plants de blé a maturité.
```
Comment (Ekdohibs) [inline locale/fr.txt]: à

File: locale/fr.txt [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+Harvest 25 fully grown wheat plants. = Récolte 25 plants de blé a maturité. 
+Harvest 625 fully grown wheat plants. = Récolte 625 plants de blé a maturité.
+Harvest a fully grown wheat plant. = Récolte un plant de blé a maturité.
+Hotelier = Hotelière/Hotelier
+House of Obsidian = La Maison de L'Obsidienne
+In the Dungeon = Dans le Dongeon
+Industrial Age = Age Industriel
```
Comment (Ekdohibs) [inline locale/fr.txt]: Âge

File: locale/fr.txt [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+Grasslands Discoverer = Découvreuse/Découvreur de prairies
+Hardened Miner = Mineuse/Mineur Endurci.e.
+Hardest Block on Earth = Le Bloc le Plus Dur du Monde
+Harvest 125 fully grown wheat plants. = Récolte 125 plants de blé a maturité.
+Harvest 25 fully grown wheat plants. = Récolte 25 plants de blé a maturité. 
+Harvest 625 fully grown wheat plants. = Récolte 625 plants de blé a maturité.
+Harvest a fully grown wheat plant. = Récolte un plant de blé a maturité.
+Hotelier = Hotelière/Hotelier
+House of Obsidian = La Maison de L'Obsidienne
+In the Dungeon = Dans le Dongeon
```
Comment (Ekdohibs) [inline locale/fr.txt]: Donjon

File: locale/fr.txt [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+Industrial Age = Age Industriel
+Join the game %d times. = Connecte toi au jeu %d fois.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Connecte-toi (toujours si on garde l'impératif, sinon, une tournure comme "se connecter").

File: locale/fr.txt [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+Join the game. = Connecte toi au jeu.
+Jungle Discoverer = Explorateur de la Jungle
```
Comment (Ekdohibs) [inline locale/fr.txt]: Explorat·eur·rice

File: locale/fr.txt [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+Junglebaby = Enfant de la Jungle
+Jungleman = Femme/Homme de la Jungle
```
Comment (Ekdohibs) [inline locale/fr.txt]: Personne de la Jungle

File: locale/fr.txt [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+Lava Miner = Mineuse/Mineur de lave
+Lava and Water = La Lave et l'Eau
+Light It Up = Eclairé.e.
+List awards in chat (deprecated) = Liste les trophés dans le chat (déprecié)
```
Comment (Ekdohibs) [inline locale/fr.txt]: (obsolète)

File: locale/fr.txt [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+Little Library = Petite Bibliothèque
+Long Ladder = Longue Echelle
+Lumberjack = Bucheron.ne.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Bûcheron·ne

File: locale/fr.txt [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+Marchand De Sable = Marchand De Sable
```
Comment (Ekdohibs) [inline locale/fr.txt]: Marchand·e de Sable

File: locale/fr.txt [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+Master Miner = Maitresse Mineuse / Maitre Mineur
```
Comment (Ekdohibs) [inline locale/fr.txt]: Maître·sse Mineu·r·se

File: locale/fr.txt [hunk 18]
```diff
@@ -0,0 +1,196 @@
...
+Mese Mastery = Maitrise de Mese
```
Comment (Ekdohibs) [inline locale/fr.txt]: Maîtrise

File: locale/fr.txt [hunk 19]
```diff
@@ -0,0 +1,196 @@
...
+Mine 18 diamond ores. = Mine 18 diamants.
+Mine 45 gold ores. = Mine 45 minerais d'or.
+Mine 50 obsidian. = Mine 50 pierres d'obsidienne.
+Mine a block: %s = Mine un bloc : %s
+Mine a mese block. = Mine un bloc de mese.
+Mine a mossy cobblestone. = Mine un bloc de pavés moussus.
+Mine a nyan cat. = Mine un nyan cat.
+Mine any block while being very close to lava. = Mine n'importe quel bloc a proximité de la lave.
```
Comment (Ekdohibs) [inline locale/fr.txt]: à

File: locale/fr.txt [hunk 20]
```diff
@@ -0,0 +1,196 @@
...
+Place 100 torches. = Place 100 torches.
+Place 2 trap stones. = Place 2 pièges de pierre.
+Place 20 coal checkers. = Place 20 pions de charbon.
+Place 20 iron checkers. = Place 20 pions de fer.
+Place 40 steel ladders. = Place 40 échelles métalliques.
+Place 400 wooden ladders. = Place 400 échelles en bois.
+Place a block: %s = Place un bloc : %s
+Place blocks: %d×%s = Place des blocs : %d×%s
+Place two snow blocks. = Place deux blocs de neige.
+Professional Lumberjack = Bucheron.ne. Professionnel.le.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Bûcheron·ne Professionnel·le

File: locale/fr.txt [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+Put out 1000 fires. = Eteint 1000 feux.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Éteint

File: locale/fr.txt [hunk 22]
```diff
@@ -0,0 +1,196 @@
...
+Pyromaniac = Pyromane
+Really Well Lit = Vraiment Bien Eclairé
```
Comment (Ekdohibs) [inline locale/fr.txt]: Éclairé

File: locale/fr.txt [hunk 23]
```diff
@@ -0,0 +1,196 @@
...
+Secret Achievement Unlocked! = Réussite Secrète Débloquée !
+Secret Achievement Unlocked: = Réussite Secrète Débloquée :
+Secret Achievement Unlocked: %s = Réussite Secrète Débloquée : %s
+Semi-pro Lumberjack = Bucheron.ne. Semi-Professionnel.le.
+Show details of an achievement = Afficher les détails d'une réussite
+Show, clear, disable or enable your achievements = Affiche, réinitialise, désactive ou active vos réussites.
+Smelter = Fondeuse/Fondeur
+Treasurer = Trésorière/Trésorier
+Unlock this award to find out what it is. = Débloque ce trophé pour découvrir ce que c'est.
+Very Simple Snow Man = Bonhome de Neige Simplissime
```
Comment (Ekdohibs) [inline locale/fr.txt]: Bonhomme

File: locale/fr.txt [hunk 24]
```diff
@@ -0,0 +1,196 @@
...
+Watchtower = Tour de Garde
+Well Lit = Bien Eclairé
+Wheat Magnate = Magnat.e. du Blé
+White Color Stock = Chaussettes Blanches
+Wool Over Your Eyes = De la Laines dans Tes Yeux
+Wow, I am Diamonds! = Wow, Je suis Diamants!
+Write %d chat messages. = Ecrit %d message dans le chat.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Écrits %d messages dans le chat.

File: locale/fr.txt [hunk 25]
```diff
@@ -0,0 +1,196 @@
...
+Write something in chat. = Ecrit quelquechose dans le chat.
```
Comment (Ekdohibs) [inline locale/fr.txt]: Écrits quelque chose dans le chat.

File: locale/fr.txt [hunk 26]
```diff
@@ -0,0 +1,196 @@
...
+You have disabled your achievements. = Vous avez désactivé vos réussites.
+You have enabled your achievements. = Vous avez activé vos réussites.
+You have not unlocked any awards. = Vous n'avez encore débloqué aucun trophé.
+You've disabled awards. Type /awards enable to reenable. = Vous avez désactivé les trophés. Tapez '/awards enable' pour les réactiver.
+You’re a copper = Faites Jouer les Cuivres
+[c|clear|disable|enable] = [c|clear|disable|enable]
+OK = OK
+Error: No awards available. = Erreur : Aucun trophé disponible
+Eat: %d×%s = Mange: %d×%s
```
Comment (Ekdohibs) [inline locale/fr.txt]: Mange : %d×%s

File: locale/fr.txt [hunk 27]
```diff
@@ -0,0 +1,196 @@
...
+Eat: %s = Mange: %s
+%d/%d eaten = %d/%d Mangés
+Place %d block(s). = Place %d bloc(s).
+Dig %d block(s). = Creuse %d bloc(s).
+Eat %d item(s). = Mange %d chose(s).
+Craft %d item(s). = Craft %d objet(s).
```
Comment (Ekdohibs) [inline locale/fr.txt]: Confectionne pour la cohérence

File: locale/fr.txt [hunk 28]
```diff
@@ -0,0 +1,196 @@
...
+Yummy! = Miam!
```
Comment (Ekdohibs) [inline locale/fr.txt]: Miam !

File: locale/fr.txt [hunk 29]
```diff
@@ -0,0 +1,196 @@
...
+Baker = Boulanger
```
Comment (Ekdohibs) [inline locale/fr.txt]: Boulang·er·ère

File: locale/fr.txt [hunk 30]
```diff
@@ -0,0 +1,196 @@
...
+Eat 10 loaves of bread. = Mange 10 miches de pain.
+Eat 80 apples. = Mange 80 pommes.
+Tasty Mushrooms = Délicieux Champignons 
+Mushroom Lover = Champignons Magiques
```
Comment (Ekdohibs) [inline locale/fr.txt]: Adorat·eur·rice de champignons

**Comment by xisd** [comment]
Merci pour la relecture !
Ça ne concerne peut être que moi, mais les accents sur autres que é et è (àûÉÈ...) ne s'affichent pas dans le jeu. 
Et, pour l'instant, je n'ai pas trouvé d'équivalent satisfaisant en français à l'expression "girl's best friend" 
Pour le reste j'ai corrigé tout ce que j'ai vu, j'espère n'avoir rien manqué. 

Thanks for the review !
This may apply only for me but accent others than é and è (àûÉÈ...) are not displayed in-game.
And I have not found a satisfying equivalent to the expression  "girl's best friend" yet.
Appart from that, I have corrected everything I've noticed, I hope I haven't missed anything.

**Comment by Ekdohibs** [comment]
About accentuated letters, what version of minetest are you using? If you are using the lastest version and still have problems, please open an issue on https://github.com/minetest/minetest/ , and I will approve this pull request.
If not, try to update your version and check if there are no more problems: if so, please use the accentuated letters in the translation.

**Comment by xisd** [comment]
You're right about accents ! I hadn't re-checked since I updgarded to 0.4.15
I added missing accents everywhere I could find them.

n.b. :  Do you happen to know where I can read the changelog of minetest 0.4.15

**Comment by sofar** [comment]
One more untranslated string. Can you fix?

**Comment by kaeza** [comment]
I have rebased this and converted to the new catalog format.

Please review #40.

**Pull request closed without merging**